components/Footer/Footer.module.scss

```diff
@@ -1,4 +1,13 @@
+.footerContainer {
+    background-color: #b3d689;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    padding: 5rem 7.5%;
```

Comment:
I think top-bottom padding could be brought down slightly (maybe closer to 2-3 rem).